share/ext/luajit/build_luajit.sh

```diff
@@ -206,7 +206,6 @@ case $1 in
 			echo "****************************************************"
 			file $PREFIX/bin/$CONF_TARGET/luajit-${DEFOLD_ARCH}
 			echo "****************************************************"
-			exit 1
```

Comment:
I didn't get why it's here.

Reply:
probably just some left over debug thing